modules/op-about-pipelinerun.adoc

```diff
@@ -48,4 +48,4 @@ spec:
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../pipelines/authenticating-pipelines-using-git-secret.adoc#authenticating-pipelines-using-git-secret[Authenticating pipelines using git secret]
+* xref:../secure/authenticating-pipelines-using-git-secret.adoc#authenticating-pipelines-using-git-secret[Authenticating pipelines using git secret]
```

Comment:
@mramendi No xrefs in modules. Refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references

Reply:
This xref was there before the restructure. The scope of this PR is limited to restructuring existing content when at all possible. Besudes this is now a standalone repository and the OpenShift requirements might not always apply, but we'll discuss with Preeti; the important part right now is to avoid scope creep, so deep restructuring is firmly out of scope.

Reply:
We might go standalone but we would want to follow uniform conventions across the board, please. A lot of content can be reused across so consistency will be key. Please add a link to all the specific comments, (like: https://github.com/openshift/openshift-docs/pull/62839#discussion_r1289972098) that are unresolved in this PR to the follow up Jira so that we do not miss them.